stdlib/camlinternalFormat.ml

```diff
@@ -2477,7 +2481,7 @@ let fmt_ebb_of_string ?legacy_behavior str =
           make_padprec_fmt_ebb (get_int_pad ()) (get_prec ()) fmt_rest in
         Fmt_EBB (Int64 (iconv, pad', prec', fmt_rest'))
     | 'f' | 'e' | 'E' | 'g' | 'G' | 'F' | 'h' | 'H' ->
-      let fconv = compute_float_conv pct_ind str_ind (get_plus ())
+      let fconv = compute_float_conv pct_ind str_ind (get_plus ()) (get_hash ())
```

Comment:
I would break the line before `(get_plus ())`, rather than before `(get_space ())`, the current placement (even before the patch) has no logic.

Reply:
done